File: trabajo 2/trabajo2_datoabs.py
class Persona:
    def __init__(persona, nombre, apellido_paterno, apellido_materno):
        persona.nombre = nombre
        persona.apellido_paterno = apellido_paterno
        persona.apellido_materno = apellido_materno

    def imprimir(self):
        print(f"Nombre completo: {self.nombre} {self.apellido_paterno} {self.apellido_materno}")

def main():
    # Crear una persona usando la clase
    persona1 = Persona("Alan", "Acosta", "Maro")

    # Imprimir la información
    persona1.imprimir()

if __name__ == "__main__":
    main()

File: trabajo 2/trabajo2_datoabs.cpp
#include <iostream>
#include <string>

using namespace std;

// Definición de la clase Persona
class Persona {
private:
    string nombre;
    string apellidoPaterno;
    string apellidoMaterno;

public:
    // Constructor
    Persona(const string& nombre, const string& apPaterno, const string& apMaterno)
        : nombre(nombre), apellidoPaterno(apPaterno), apellidoMaterno(apMaterno) {}

    // Método para imprimir la información de la persona
    void imprimir() const {
        cout << "Nombre completo: " << nombre << " " << apellidoPaterno << " " << apellidoMaterno << endl;
    }
};

int main() {
    // Crear una persona usando la clase
    Persona persona1("Alan", "Acosta", "Maro");

    // Imprimir la información
    persona1.imprimir();

    return 0;
}
